src/utils/subscriberBreaches.ts

```diff
@@ -157,8 +161,28 @@ export async function getSubBreaches(
             (curBreach.dataClassesEffected[index][key] as number)++;
           }
         });
+
+        // Only mark data classes as resolved if they are resolved for all
+        // affected email addresses:
+        // We check if a data class is already resolved for the other email
+        // addresses. If that is the case, the respective data class are
+        // duplicated in `combinedResolvedDataClasses`. Those can be considered
+        // resolved and stay in the list of resolved data classes.
+        // Unique data classes will be filtered out as they haven’t been
+        // resolved for all affected email addresses.
```

Comment:
Thanks. If we don’t add any more exceptions like for `SSN` breaches it’s not too bad. In case we need to add more logic around breach resolution we might have to start that with a small refactor.